src/mlpack/methods/naive_bayes/naive_bayes_classifier.hpp

```diff
@@ -92,11 +92,14 @@ class NaiveBayesClassifier
    * Probabilities() individually to set them to the right size.
    *
    * @param data The dataset to train on.
+   * @param labels The labels for the dataset.
+   * @param numClasses The numbe of classes in the dataset.
    * @param incremental Whether or not to use the incremental algorithm for
    *      training.
    */
   void Train(const MatType& data,
              const arma::Row<size_t>& labels,
+             const size_t numClasses,
```

Comment:
Is it ok just to add a new parameter to the `Train` method? Shouldn't we deprecate the old method and add a new one?